src/movies.js

```diff
@@ -1,7 +1,15 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors() {}
+function getAllDirectors(moviesArr) {
+  const result = moviesArr.map(function(filmObj) {
+    return filmObj.director;
+  });
+ 
+  console.log(result)
```

Comment:
Do not forget: console.log and return is not the same thing. You usually want to return the result of your function

-> Console.log is a javascript function to print something in the console.
-> While return saves the result of your function in memory, so you can reuse that result or print it later.
Return by itself does not print anything